.github/workflows/dependency-review.yml

```diff
@@ -0,0 +1,20 @@
+# Dependency Review Action
+#
+# This Action will scan dependency manifest files that change as part of a Pull Request, surfacing known-vulnerable versions of the packages declared or updated in the PR. Once installed, if the workflow run is marked as required, PRs introducing known-vulnerable packages will be blocked from merging.
+#
+# Source repository: https://github.com/actions/dependency-review-action
+# Public documentation: https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-dependency-review#dependency-review-enforcement
+name: 'Dependency Review'
+on: [pull_request]
```

Comment:
```suggestion
on: 
    pull_request:
       branches: [main, alpha]
```

Reply:
what about the staging branch

Reply:
we are not making pr against the staging branch . or are we ?

Reply:
nope good catch